Description: Fix FTCBFS: hard codes the build architecture pkg-config
Author: Helmut Grohne <helmut@subdivi.de>
Debian-Bug: https://bugs.debian.org/958484
--- a/configure.ac
+++ b/configure.ac
@@ -19,7 +19,7 @@
 AC_SUBST(LIBXKLAVIER_LIBS)
 AC_SUBST(LIBWNCK_CFLAGS)
 AC_SUBST(LIBWNCK_LIBS)
-LIBXKLAVIER_VERSION=`pkg-config --modversion libxklavier | sed 's/\.//;s/\..*//g'`
+LIBXKLAVIER_VERSION=`$PKG_CONFIG --modversion libxklavier | sed 's/\.//;s/\..*//g'`
 AC_DEFINE_UNQUOTED(LIBXKLAVIER_VERSION, ${LIBXKLAVIER_VERSION}, [libxklavier version])
 
 APPINDICATOR_REQUIRED_VERSION=0.2.4
